README.md

```diff
@@ -1,8 +1,21 @@
 # This is public data
 
 ## installation
+After Successfully setting up the student-intern-organiser Application repository 
+Do the Following to setup the Database
+1. Change Directory/Folder
+```
+cd student-intern-organiser
+```
+2. clone the repository
+```
+git clone https://github.com/WEHI-ResearchComputing/student_intern_data_public.git
```

Comment:
You can do this all in one command:

git clone https://github.com/WEHI-ResearchComputing/student_intern_data_public.git student_intern_data